app/storage/dynamo_api.py

```diff
@@ -0,0 +1,71 @@
+from botocore.exceptions import ClientError
+from flask import current_app
+from structlog import get_logger
+
+logger = get_logger()
+
+
+def put_item(table_name, item, overwrite=True):
+    """Insert an item into table"""
+    table = _get_table(table_name)
+
+    put_kwargs = {'Item': item}
+    if not overwrite:
+        first_key = next(iter(item.keys()))
+        put_kwargs['ConditionExpression'] = 'attribute_not_exists({first_key})'.format(
+            first_key=first_key)
+
+    response = table.put_item(**put_kwargs)['ResponseMetadata']['HTTPStatusCode']
+    return response == 200
+
+
+def get_item(table_name, key):
+    """ Get an item given its key """
+    table = _get_table(table_name)
+
+    response = table.get_item(Key=key)
+    item = response.get('Item', None)
+
+    return item
+
+
+def delete_item(table_name, key):
+    """Deletes an item by its key
+    """
+    table = _get_table(table_name)
+
+    response = table.delete_item(Key=key)
+    item = response.get('Item', None)
+    return item
+
+
+def delete_all(table_name):
+    """Deletes all items from a table
+    """
+    table = _get_table(table_name)
+
+    try:
+        keys = [k['AttributeName'] for k in table.key_schema]
+    except ClientError as e:
+        if e.response['Error']['Code'] == 'ResourceNotFoundException':
+            return
+
+    response = table.scan()
+
+    with table.batch_writer() as batch:
+        for item in response['Items']:
+            key_dict = {k: item[k] for k in keys}
+            batch.delete_item(Key=key_dict)
+
+
+def _get_table(table_name):
+    return current_app.eq['dynamodb'].Table(table_name)
+
+
+def create_table(table_name, key_field):
+    current_app.eq['dynamodb'].create_table(
```

Comment:
We shouldn't be creating tables in code.
It also wont/shouldn't have permission to do this

Reply:
This is to create tables in the dynamodb mock, I'm using moto